wormhole-connect/src/hooks/useRoutesQuotesBulk.ts

```diff
@@ -32,9 +32,7 @@ type HookReturn = {
 };
 
 const QUOTE_REFRESH_INTERVAL = 20_000;
-
-const MAYAN_BETA_LIMIT = 10_000; // USD
-const MAYAN_BETA_PROTOCOLS = ['MCTP', 'SWIFT'];
+const MAYAN_BETA_PROTOCOL_LIMITS = { MCTP: 10_000, SWIFT: null };
```

Comment:
just remove `SWIFT` from this object. there's a third protocol `'WH'` which also doesn't have a limit.

https://github.com/mayan-finance/wormhole-sdk-route/blob/main/src/index.ts#L525